examples/resources/leanspace_leaf_space_contact_reservations.tf

```diff
@@ -1,4 +1,4 @@
-resource "leanspace_leaf_space_contact_reservations" "contact_reservation" {
+resource "leanspace_leaf_space_contact_reservations" "contact_reservation_status_mapping" {
```

Comment:
the resource should be renamed as well as the file also I expect mapping to be plural

Reply:
Ok done.